### PR TITLE
[Validator] Add Hungarian translations

### PR DESCRIPTION
**Pull request by 1ed**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf [hunk 1]
```diff
@@ -366,6 +366,22 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Ennek az értéknek {{ min }} és {{ max }} között kell lennie.</target>
             </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Ez az érték nem érvényes állomásnév (hosztnév).</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>A gyűjetemény elemszámának oszthatónak kell lennie a következővel: {{ compared_value }}.</target>
```
Comment (paradajozsef) [inline src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf]: `gyűjtemény`

**Review by paradajozsef — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf (hunk 1 above)
Comment (gyszucs) [inline src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf]: A typo here, it should be "gyűjtemény". Anyway, how about starting this way: "A gyűjteményben lévő elemek számának..."

**Review by gyszucs — COMMENTED** [review]
Thanks for the PR, I just added two minor comments.

File: src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf (hunk 1 above)
Comment (1ed) [reply src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf]: Thx

Comment (1ed) [reply src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf]: Thanks, it's better this way.

**Review by paradajozsef — APPROVED** [review]

**Review by gyszucs — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @1ed.

**Pull request closed without merging**